Gemfile

```diff
@@ -2,6 +2,9 @@ source "https://rubygems.org"
 gemspec :name => "jekyll"
 
 gem "rake", "~> 11.0"
+
+gem "activesupport", "~> 4.2", :groups => [:test_legacy, :site] if RUBY_ENGINE == 'ruby' && RUBY_VERSION < '2.2.2'
```

Comment:
@parkr Good question, i used Rails' [ruby_version_check.rb](https://github.com/rails/rails/blob/52ce6ece8c8f74064bb64e0a0b1ddd83092718e1/railties/lib/rails/ruby_version_check.rb) as an example but could be possible to leave it out i guess.